support-frontend/assets/helpers/forms/existingPaymentMethods/existingPaymentMethods.ts

```diff
@@ -34,7 +40,11 @@ export type ExistingPaymentMethod =
 function isUsableExistingPaymentMethod(
 	existingPaymentMethod: ExistingPaymentMethod,
 ): boolean {
-	return !!existingPaymentMethod.billingAccountId;
+	if ('billingAccountId' in existingPaymentMethod) {
+		return !!existingPaymentMethod.billingAccountId;
+	} else {
```

Comment:
We could do away with the `else` block here and simply `return false` after the `if` block.

Reply:
Yeah that makes sense, or maybe 

```ts
('billingAccountId' in existingPaymentMethod) && !!existingPaymentMethod.billingAccountId
```

But that might be a little hard to read